xml/tuning_cgroups.xml

```diff
@@ -57,6 +57,30 @@
   </para>
  </sect1>
 
+ <sect1 xml:id="sec-tuning-cgroups-accounting">
+  <title>Resource accounting</title>
+  <para>
+  The placement of processes into different cgroups can be used to obtain
+  per-cgroup information of certain resource consumptions.
+  </para>
+  <para>
+  The accounting has relatively small but non-zero overhead whose impact
+  depends on the workload.
+  Be aware that turning on accounting for one unit will also implicitly turn it
+  on for all units directly contained in the same slice and for all its parent
+  slices and the units directly contained therein.
+  Therefore the accounting cost is not exclusive to the single unit.
+  </para>
+  <para>
+  The accounting can be set on per-unit basis with directives such as
+  <literal>MemoryAccounting=</literal> or globally for all units in
+  <filename>/etc/systemd/system.conf</filename> with respective directive
```

Comment:
I would refrain from mentioning /etc/systemd/system.conf or any other main config files. Those files can be overridden by drop-ins (wherever they're located) and thus be overridden by the drop-ins we (downstream) ship to customize upstream defaults.

This behavior is documented in systemd documentation but it's quite unexpected by users. I still need to clarify whether this is really intended (see https://github.com/systemd/systemd/pull/17161) and if upstream decides to keep this behavior then we will probably extend our documentation by explaining that customers settings should be done via drop-ins (with higher priority than the one we use).

Reply:
Thank you @fbuihuu, that is an important clarification. I recall looking at this before, and we need to get it right in the docs. I will review the docs to see if anything needs updating.